.travis.yml

```diff
@@ -121,20 +121,16 @@ jobs:
     env: DESC="Deploy server Docker image"
     script:
     - $TRAVIS_BUILD_DIR/scripts/travis/deploy_docker.sh
-    # We temporarily disable the deployment of client to Qiniu because the
-    # qshell upload feature is unstable and often fails the CI, c.f.,
-    # https://github.com/sql-machine-learning/sqlflow/issues/2305
-    #
-    # - env: DESC="Deploy macOS client"
-    #   os: osx
-    #   script:
-    #   - $TRAVIS_BUILD_DIR/scripts/travis/deploy_client.sh
-    # - env: DESC="Deploy Linux client"
-    #   os: linux
-    #   dist: bionic
-    #   script:
-    #   - $TRAVIS_BUILD_DIR/scripts/travis/deploy_client.sh
-    # - env: DESC="Deploy Windows client"
-    #   os: windows
-    #   script:
-    #   - $TRAVIS_BUILD_DIR/scripts/travis/deploy_client.sh
+      - env: DESC="Deploy macOS client"
```

Comment:
The indentation looks wrong. The line `-env: DESC="Deploy macOS client"` should be on the same identation level as `env: DESC="Deploy server Docker image"`?

Reply:
Done